universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -32,6 +32,18 @@
 import services.applicant.predicate.JsonPathPredicate;
 import services.applicant.question.Scalar;
 
+/**
+ * Brokers access to the answer data for a specific applicant at a specific version.
```

Comment:
I don't think `ApplicantData` knows about versions. It represents the latest answers of an applicant. If they haven't checked back in a long time, then the applicant data is old, but we would try to use it with the live data.

There was an idea at some point to give admins the option to clear answers to a question when they publish a new version. This would go into the single copy of applicant data and wipe its answers.